tests/flux/test_flux_speed.py

```diff
@@ -0,0 +1,64 @@
+import logging
+import time
+
+import pytest
+import torch
+from diffusers import FluxPipeline
+
+from nunchaku import NunchakuFluxTransformer2dModel, NunchakuT5EncoderModel
+from nunchaku.utils import get_precision, is_turing
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@pytest.mark.skipif(is_turing(), reason="Skip tests due to using Turing GPUs")
+@pytest.mark.parametrize(
+    "warmup_times,test_times,num_inference_steps,guidance_scale,use_qencoder,expected_latency",
+    [
+        (2, 5, 30, 3.5, True, 6.49650 if get_precision() == "int4" else 4.79388),
```

Comment:
Do not use precision to get the latency. Please use the specific GPU. If the GPU is not in the list, just skip the test.